### ISSUE TITLE
how to compress in my dataset

### ISSUE THREAD

**Issue by as754770178** [Tencent/PocketFlow#51]
When I compress model on my custom dataset, the function `def setup_lrn_rate(global_step, model_name, dataset_name)` raise the NotImplement error. I want to produce a function that return lr and batch numbers in normal dataset and normal module. How I set the value of `nb_epoches`, `decay_rates` ...? How to choose piecewise constant strategy or exponential decaying strategy?

**Comment by jiaxiang-wu** [comment]
You can set these hyper-parameter similar to how you train a normal/uncompressed model on your own data set, without PocketFlow.

**Comment by as754770178** [comment]
Can you give me an example? Which parameters should I set?

**Comment by jiaxiang-wu** [comment]
Before PocketFlow, have you trained some models on this data set? What is your hyper-parameter setting, or, how do you adjust the learning rate during the training process at that time?

**Comment by as754770178** [comment]
I mean that can I set some hyper-parameter so that PocketFlow does not call `setup_lrn_rate` function?

**Comment by jiaxiang-wu** [comment]
Almost all model compression components need this function to setup the **learning rate schedule** and **number of training iterations**. I suggest you do not skip this function. If you do want to skip this, you should modify the corresponding model compression component, and manually setup the learning rate schedule.

**Comment by as754770178** [comment]
OK，thanks
